### PR TITLE
Update tests data

### PR DESCRIPTION
**Pull request by Brivan-26**
Closes [Update goerli and goerli2 test data to 0.12.1](https://github.com/NethermindEth/juno/issues/948)

### CODE REVIEW

File: clients/feeder/feeder_test.go [hunk 1]
```diff
@@ -141,7 +141,7 @@ func TestBlockWithoutSequencerAddressUnmarshal(t *testing.T) {
 	assert.Equal(t, "0x3873ccb937f14429b169c654dda28886d2cc2d6ea17b3cff9748fe5cfdb67e0", block.ParentHash.String())
 	assert.Equal(t, uint64(11817), block.Number)
 	assert.Equal(t, "0x3df24be7b5fed6b41de08d38686b6142944119ca2a345c38793590d6804bba4", block.StateRoot.String())
-	assert.Equal(t, "ACCEPTED_ON_L2", block.Status)
+	assert.Equal(t, "ACCEPTED_ON_L1", block.Status)
```
Comment (joshklop) [inline clients/feeder/feeder_test.go:144]: We should manually modify the original json file. At the time the files were downloaded, the block was only `ACCEPTED_ON_L2`, and it's important we continue to test that code path.

File: rpc/handlers_test.go [hunk 2]
```diff
@@ -1846,7 +1846,7 @@ func TestTransactionStatus(t *testing.T) {
 		{
 			network:           utils.MAINNET,
 			verifiedTxHash:    utils.HexToFelt(t, "0xf1d99fb97509e0dfc425ddc2a8c5398b74231658ca58b6f8da92f39cb739e"),
-			nonVerifiedTxHash: utils.HexToFelt(t, "0x6c40890743aa220b10e5ee68cef694c5c23cc2defd0dbdf5546e687f9982ab1"),
+			nonVerifiedTxHash: utils.HexToFelt(t, "0x6cd1526afa9e294bfccf58e0fc73f0486175bbfed11d6bf661c129d3cc8b9f0"),
```
Comment (joshklop) [inline rpc/handlers_test.go:1849]: Can you please provide some context for this change?

**Review by joshklop — APPROVED** [review]
LGTM. I didn't go through each json file, but if the tests pass I trust it's done correctly :)

File: rpc/handlers_test.go (hunk 2 above)
Comment (Brivan-26) [reply rpc/handlers_test.go:1849]: Oops, I was just testing something locally, and forgot to reset it back.

File: clients/feeder/feeder_test.go [hunk 3]
```diff
@@ -159,7 +159,7 @@ func TestBlockWithSequencerAddressUnmarshal(t *testing.T) {
 	assert.Equal(t, "0x68427fb6f1f5e687fbd779b3cc0d4ee31b49575ed0f8c749f827e4a45611efc", block.ParentHash.String())
 	assert.Equal(t, uint64(19199), block.Number)
 	assert.Equal(t, "0x541b796ea02703d02ff31459815f65f410ceefe80a4e3499f7ef9ccc36d26ee", block.StateRoot.String())
-	assert.Equal(t, "ACCEPTED_ON_L2", block.Status)
+	assert.Equal(t, "ACCEPTED_ON_L1", block.Status)
```
Comment (joshklop) [inline clients/feeder/feeder_test.go:162]: This should still be ACCEPTED_ON_L2.

**Review by joshklop — CHANGES_REQUESTED** [review]

**Review by joshklop — APPROVED** [review]

**Pull request merged**